lib/modules/apostrophe-favicons-global/index.js

```diff
@@ -201,9 +196,14 @@ the global preferences are saved.`,
               }
             });
             const sizeExtension = size ? (size.name + '.') : '';
+
             let originalPath = '/attachments/' + image.attachment._id + '-' + image.attachment.name + '.' + sizeExtension + image.attachment.extension;
```

Comment:
How about using the uploadfsPath: true option to attachments.url so you don't have to reinvent this and risk any inconsistency in future?

Reply:
With that option attachments.url gives you an uploadfs path instead of a proper URL.